src/deadline/client/cli/_incremental_download.py

```
@@ -368,6 +368,13 @@ def _categorize_jobs_in_checkpoint(
 
         print_function_callback(f"NEW Job: {dc_job['name']} ({job_id})")
 
+        if dc_job["storageProfileId"] is None and checkpoint.local_storage_profile_id is not None:
+            print_function_callback(
```

One thing we discussed during ORR was to add a regex print in the logs for such jobs for the customer to easily filter them out of the console/cron logs in case they get into an issue. This is specifically important for the case where there's a file system location name missing for a job storage profile (so a misconfigured storage profile not a missing one).

---

I think you're talking about this warning message, is that right?

https://github.com/aws-deadline/deadline-cloud/blob/mainline/src/deadline/client/cli/_incremental_download.py#L1113

---

oh yes this looks like it. Do you mind adding some information about the unmapped paths too? Might be useful for them to debug - I'm just looking at it from the operational lens for a customer if they need to re-run some failed stuff because we aren't failing fast & continue downloading the rest if there's an unmapped path?

---

Yes, makes sense. Will have to think about where we put that, whether it's documentation or also goes somewhere else.